mitodl/ocw-to-hugo
# upgrade turndown

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

updates turndown, which was recently updated to no longer depend on JSDom, instead using [domino](https://github.com/fgnass/domino), which is much faster. ocw-to-hugo now runs in around half the time it used to.

I ran 
```
time node ./src/bin/index.js -c course_json_examples/example_courses.json -i private/courses -o private/output --strips3`
```

to test. on master:

```
62.59s user 3.83s system 127% cpu 52.244 total
```

with turndown v7.0.1:

```
25.27s user 2.89s system 125% cpu 22.380 total
```

#### How should this be manually tested?

we need to ensure that this change doesn't introduce any problems into ocw-to-hugo's output. I have run the example courses through and those render in hugo-course-publisher without any errors or (apparent) issues. I also diffed the output of those courses with the output on ocw-to-hugo master and there are a few differences but nothing that I think will be problematic.

but as far as testing this I would check that!
